mmv1/products/sql/SourceRepresentationInstance.yaml

```diff
@@ -80,8 +80,6 @@ properties:
         description: |
           The externally accessible IPv4 address for the source database server.
```

Comment:
Could you please update the description to reflect the information you included in the PR description?

From the documentation you linked to, this looks like a good replacement for the current description in the provider:

SOURCE HOST | The IPv4 address and port for the external server, or the the DNS address for the external server. If the external server is hosted on Cloud SQL, the port is 5432.
-- | --